sys/include/net/gnrc/gnrc_mac/packet_queue.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2015 Daniel Krebs
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ * @ingroup     net_gnrc_mac
+ * @file
+ * @brief       Wrapper for priority_queue that holds gnrc_pktsnip_t* and is
```

Comment:
What's wrong with [gnrc_pktqueue](http://doc.riot-os.org/group__net__gnrc__pktqueue.html) that you introduce another packet queue?